src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -174,18 +175,78 @@ private static TryCatchStatement composePagedUnaryRpcMethodSampleCode(
   }
 
   private static TryCatchStatement composeUnaryRpcDefaultMethodSampleCode(
-      Method method, TypeNode clientType) {
+      Method method, TypeNode clientType, Map<String, ResourceName> resourceNames) {
     // TODO(summerji): compose sample code for unary default rpc method.
     // TODO(summerji): Add unit tests.
-    String content =
-        String.format(
-            "Note: Not Implement yet, placeholder for unary %s rpc method sample code.",
-            (!method.hasLro() && !method.isPaged()
-                ? "default"
-                : (method.hasLro() ? "lro default" : "paged default")));
+    // If variant method signatures exists, use the first one.
+    List<MethodArgument> arguments =
+        !method.methodSignatures().isEmpty()
+            ? method.methodSignatures().get(0)
+            : Collections.emptyList();
+    // Assign each method arguments with default value.
+    List<Statement> bodyStatements =
+        arguments.stream()
+            .map(
+                methodArg ->
+                    ExprStatement.withExpr(
+                        assignMethodArgumentWithDefaultValue(methodArg, resourceNames)))
+            .collect(Collectors.toList());
+    // Assign request variables with set argument attributes.
+    // e.g EchoRequest
+    bodyStatements.add(
+        ExprStatement.withExpr(createRequestBuilderExpr(method.inputType(), arguments)));
+
+    if (method.isPaged()) {
+      // For loop on invoke client method's iterator with comment.
+      // e.g. for (EchoClient echoClient : echoClient.PagedExpand(request).iterateAll()) {//..}
```

Comment:
This line looks off, I'm not sure that `iterateAll` on that method would return a list of `echoClient`s.